src/Microsoft.Health.Dicom.Core/Features/Common/DicomTagParser.cs

```diff
@@ -39,6 +39,20 @@ public bool TryParse(string dicomTagPath, out DicomTag[] dicomTags, bool support
             return dicomTag != null;
         }
 
+        private static DicomTag ParseDicomTagKeyword(string keyword)
```

Comment:
Should the method indicate somehow that only public tags are allowed? Like `ParsePublicDicomTagKeyword`?

Reply:
sure, might be ParseStandardDicomTagKeyword as normally called Standard instead of Public http://dicom.nema.org/dicom/2013/output/chtml/part05/sect_7.8.html